docs/module_types/bob_transform_source.md

```diff
@@ -31,9 +31,9 @@ bob_transform_source {
     out: {
         match: "file_([0-9])+.cpp",
         replace: ["new_$1.o"],
-        depfile: "$1.d",
         implicit_srcs: ["my_file.scu"],
     },
+    depfile: true,
```

Comment:
Can we add a `${depfile}` reference to `cmd` in this documentation? We require that if depfile is true so it would be useful to mention it.